modules/core/Cockpit/assets/components/field-html.tag

```diff
@@ -37,7 +37,14 @@
             ], function() {
 
                 $this.input.value = $this.value;
-
+                //pass along codemirror options to htmleditor/codemirror
+                if(opts.codemirroroptions){
+                    co=opts.codemirroroptions;
```

Comment:
This line should be `var co = opts.codemirroroptions;`, otherwise it's leaking into global scope